src/main/java/com/epam/tc/hw9/assertations/ListAssertions.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.tc.hw9.assertations;
+
+import static com.epam.tc.hw9.constans.Endpoints.LISTS;
+import static org.testng.Assert.assertEquals;
+
+import com.epam.tc.hw9.dto.board.BoardDTO;
+import com.epam.tc.hw9.dto.list.ListDTO;
+import com.epam.tc.hw9.services.CommonService;
+import com.epam.tc.hw9.services.RestTrelloService;
+import com.epam.tc.hw9.utils.GetPropertyFile;
+import io.restassured.response.Response;
+import java.util.Map;
+
+public class ListAssertions {
+    RestTrelloService restTrelloService = new RestTrelloService();
+
+    public void assertListNotExist(ListDTO listDTO) {
+        Response response = restTrelloService.getListResponse(listDTO.getId());
+        assertEquals(response.asPrettyString(), "model not found", String.format("Response does not match what is expected"
+            + " (\"model not found\"), see response: %s", response.asPrettyString()));
+    }
+
+    public void assertCreatingOnDeletedBoard(Map<String, String> params, BoardDTO boardDTO) {
+        params.put("name", (GetPropertyFile.getProperty("src/test/resources/testHw9.properties")
```

Comment:
????
create separate class for property reading.
and no not supply file name each time

I ideal world file is read once